scripts/deploy_assisted_service.sh

```diff
@@ -67,8 +68,10 @@ if [ "${OPENSHIFT_INSTALL_RELEASE_IMAGE}" != "" ]; then
     fi
 fi
 
-OS_IMAGES=$(skipper run ./scripts/override_os_images.py --src ./assisted-service/data/default_os_images.json)
-export OS_IMAGES
+if [ "${OPENSHIFT_CI}" == "true" ]; then
+    OS_IMAGES=$(skipper run ./scripts/override_os_images.py --src ./assisted-service/data/default_os_images.json)
```

Comment:
just nit may this should be in a variable for changes or overwrites
```suggestion
    IMAGES_JSON=' ./assisted-service/data/default_os_images.json'
    OS_IMAGES=$(skipper run ./scripts/override_os_images.py --src ${IMAGES_JSON})
```

also can you add in `override_os_images.py` validattion check and error output before the file read
```python
"""  here  check if file readable """
   with open(args.src, "r") as f:
        os_images: list = json.load(f)

```

Reply:
Also I don't think this is necessary (at least to run again all of the tests for it), because default_os_images.json always exists